reaktive/src/commonMain/kotlin/com/badoo/reaktive/utils/PrintError.kt

```diff
@@ -1,3 +1,14 @@
 package com.badoo.reaktive.utils
 
+import com.badoo.reaktive.utils.atomic.AtomicBoolean
+
+@Suppress("ObjectPropertyName") // Backing field
+private val _isPrintErrorEnabled = AtomicBoolean(true)
+
+internal var isPrintErrorEnabled
```

Comment:
Use it for `printStack()` too. It is quite strange, that you can skip some logging of small messages like `Some shit happened` but not huge stacktraces.

Maybe in future we should introduce `Logger` that can log text and exceptions (with stacktraces) and default implementation of it in the way it is implemented now.
For example on Android it will become possible to use `Tinder` to log it and report to server.

Reply:
Let's keep it as is for now, but introduce `Logger ` interface later, so we can mock logger for android.